hmrc-mongo-play-27/src/test/scala/uk/gov/hmrc/mongo/play/json/PlayMongoRepositorySpec.scala

```diff
@@ -395,4 +427,19 @@ object PlayMongoRepositorySpec {
       listString        = ls,
       listLong          = ll
     )
+
+  val flatJsonObjectGen: Gen[JsObject] = {
+    // Not all Strings are valid bson field named (e.g. with `.`)
+    case class BsonFieldName(s: String)
+    implicit val bsonFieldNameGen: Arbitrary[BsonFieldName] =
+      Arbitrary(
+        Arbitrary.arbitrary[String]
```

Comment:
Can we use alphanumeric characters here? At the moment it is generating 
```
List("荆䟁靆䕺ࡻ", "ﾎ", "⎫퓘곭뙢퇢紩陵쭩", "≞䪢ﰵ䎗┌᫻杈", "遜䈄룴큖ٍኈ", "⏳⅛ᯙ揗ࣽ㧍苵", "樎샕쏍麬亨ἁ哆", "鯣Ǻฒ") was not equal to List("≞䪢ﰵ䎗┌᫻杈", "⎫퓘곭뙢퇢紩陵쭩", "樎샕쏍麬亨ἁ哆", "遜䈄룴큖ٍኈ", "荆䟁靆䕺ࡻ", "鯣Ǻฒ", "⏳⅛ᯙ揗ࣽ㧍苵", "_id")
```

This can be hard to debug tests.